indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -727,3 +727,10 @@ form.horizontal .clone-event-dialog-footer .form-field {
         margin-left: auto;
     }
 }
+
+
+.event-locked {
+    .hide-if-locked {
+        display: none !important;
```

Comment:
Yes, but the whole point of this is to conditionally hide things if a parent has `.event-locked`.